gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/MysqlDagActionStore.java

```diff
@@ -163,17 +163,17 @@ public boolean deleteDagAction(DagAction dagAction) throws IOException {
   }
 
   // TODO: later change this to getDagActions relating to a particular flow execution if it makes sense
-  private DagAction getDagActionWithRetry(String flowGroup, String flowName, String flowExecutionId, String jobName, DagActionType dagActionType, ExponentialBackoff exponentialBackoff)
+  private DagAction getDagActionWithRetry(String flowGroup, String flowName, long flowExecutionId, String jobName, DagActionType dagActionType, ExponentialBackoff exponentialBackoff)
       throws IOException, SQLException {
     return dbStatementExecutor.withPreparedStatement(String.format(GET_STATEMENT, tableName), getStatement -> {
       int i = 0;
       getStatement.setString(++i, flowGroup);
       getStatement.setString(++i, flowName);
-      getStatement.setString(++i, flowExecutionId);
+      getStatement.setString(++i, String.valueOf(flowExecutionId));
       getStatement.setString(++i, dagActionType.toString());
       try (ResultSet rs = getStatement.executeQuery()) {
         if (rs.next()) {
-          return new DagAction(rs.getString(1), rs.getString(2), rs.getString(3), rs.getString(4), DagActionType.valueOf(rs.getString(5)));
+          return new DagAction(rs.getString(1), rs.getString(2), rs.getLong(3), rs.getString(4), DagActionType.valueOf(rs.getString(5)));
```

Comment:
does the JDBC `ResultSet` allow this?  I thought it might be necessary to do:
```
Long.valueOf(rs.getString(3))
```

Reply:
online resources say that it does the conversion itseld.